system/system.go

```diff
@@ -48,14 +48,17 @@ func (s *System) Ports() map[string][]GOnetstat.Process {
 	return s.ports
 }
 
-func (s *System) ProcMap() map[string][]ps.Process {
+func (s *System) ProcMap() (map[string][]ps.Process, error) {
+	var err error
+
 	s.procOnce.Do(func() {
-		s.procMap = GetProcs()
+		s.procMap, err = GetProcs()
 	})
-	return s.procMap
+
+	return s.procMap, err
 }
 
-func New(c *cli.Context) *System {
+func New(packageManager string) *System {
```

Comment:
I can see some benefits to passing in RuntimeConfig object to function.

This can be changed as a later PR.

Reply:
yup, however it would cause an import cycle.

We can resolve this when, towards the end, I look to merge util.Config and goss.RuntimeConfig, i held off on that to keep the PR from just going totally crazy cross many boundaries.